src/confluent_kafka/src/Admin.c

```diff
@@ -4369,6 +4470,52 @@ static PyObject *Admin_c_ListOffsetsResultInfos_to_py (const rd_kafka_ListOffset
         return NULL;
 }
 
+static PyObject *Admin_c_DeletedRecords_to_py (const rd_kafka_topic_partition_list_t *c_topic_partitions) {
+    
+    size_t c_topic_partition_cnt = c_topic_partitions->cnt;
+    PyObject *result = NULL;
+    PyObject *DeleteRecords_type = NULL;
+    size_t i;
+
+    DeleteRecords_type = cfl_PyObject_lookup("confluent_kafka", "DeleteRecords");
+ 
+    if(!DeleteRecords_type){
+        cfl_PyErr_Format(RD_KAFKA_RESP_ERR__INVALID_ARG, "Unable to load DeleteRecords type");
+        goto raise;
+    }
+
+    result = PyDict_New();
+    for(i=0; i<c_topic_partition_cnt; i++){
+        PyObject *value = NULL;
+        rd_kafka_topic_partition_t *c_topic_partition = &c_topic_partitions->elems[i];
+        if (c_topic_partitions->elems[i].err) {
```

Comment:
Why not use `c_topic_partition` everywhere in place of `c_topic_partitions->elems[i]`?